jwst/resample/resample_step.py

```diff
@@ -48,7 +48,7 @@ class ResampleStep(Step):
         output_wcs = string(default='')  # Custom output WCS
         single = boolean(default=False)  # Resample each input to its own output grid
         blendheaders = boolean(default=True)  # Blend metadata from inputs into output
-        in_memory = boolean(default=True)  # Keep images in memory
+        in_memory = boolean(default=True)  # Keep images in memory when 'single' is False
```

Comment:
Would you explain this change?

Is the comment correct? On line 105 below `in_memory` is used within an `if self.single` block where `many_to_many` (single=True) is called.

Reply:
`single=False => "many"` which is what `many_to_many` is. It is the only place in the code that uses `in_memory` (to create a model library `on_disk` or not). `many_to_one` and `resample_group` return a single model (not a library) that is always in memory.

Reply:
Is the meaning of `single` being changed with this PR? On main `single=True` results in `many_to_many`:
https://github.com/spacetelescope/jwst/blob/f7bb7b6c53752600d006f5bff1c451a1315cae50/jwst/resample/resample.py#L170-L173

`in_memory` is also used to open the `ModelLibrary` off inputs:
https://github.com/mcara/jwst/blob/b1cbbecce486f2fbb0d5089f23bcc6a52b867f99/jwst/resample/resample_step.py#L61

I think the addition to this comment is misleading as that applies for both `single=True` and `single=False`.

Reply:
You are right. I will remove comment change in a future commit.